R/utilities-sensitivity-analysis.R

```diff
@@ -662,7 +697,10 @@ plotPopulationSensitivity <- function(structureSets,
     saResultIndexFiles[[populationName]] <- structureSet$popSensitivityAnalysisResultsIndexFile
     simulationList[[populationName]] <- simulation
 
-    for (op in outputPaths) {
+    for (output in structureSet$simulationSet$outputs) {
+      op <- output$path

```

Comment:
Rename to outputPath if it's needed at all

Reply:
Fixed